cognitev/eventful-django
# RespectEventfulBrokerBackendURLs

**Description:**
This PR is created to respect eventful envs even if celery envs if exists
**JIRA:**
https://menacommerce.atlassian.net/browse/INS-4718


**Testing instructions:**

1. Schedule a task on eventful worker (like eventful worker)
2- Check if eventful worker pull the task and handle it correctly or not

**Reviewers:**
@AlaaNour94 
@AlSayedGamal 
@Kareem-Emad 
@amr-alaa95 

**Merge checklist:**

**Post merge:**
      finished.
